src/sprout/core/Kohana.php

```diff
@@ -692,6 +698,25 @@ public static function errorHandler($errno, $errmsg, $file, $line)
         throw new ErrorException($errmsg, 0, $errno, $file, $line);
     }
 
+
+    public static function handleFatalErrors()
```

Comment:
Could be pedantic about a docblock here, but the name is self evident :D

Reply:
No, please!

I'll add one now.